tests/ProxyManagerTest/Functional/LazyLoadingGhostFunctionalTest.php

```diff
@@ -1516,6 +1520,47 @@ static function (
         self::assertSame($initialCounter + $increment, $proxy->counter);
     }
 
+    /**
+     * @group 723
+     * @psalm-suppress UnusedVariable this method uses by-ref assignment of properties, and isn't recognized by static analysis
+     * @psalm-suppress UndefinedClass Class, interface or enum named never does not exist
+     */
+    public function testWillExecuteLogicInANeverMethod(): void
+    {
+        if (PHP_VERSION_ID < 80100) {
+            self::markTestSkipped('Needs PHP 8.1');
+        }
```

Comment:
Can be replace by annotation:
```php
/** @requires PHP >= 8.1 */
```

Reply:
Similar checks have been used in other tests, but yes could be replaced easily if needed.